src/v0/util/tags.js

```diff
@@ -52,6 +52,7 @@ const ERROR_TYPES = {
   UNSUPPORTED: 'unsupported',
   REDIS: 'redis',
   FILTERED: 'filtered',
+  UNAUTHORIZED: 'accessTokenExpired',
```

Comment:
import `ERROR_TYPES` from `integrations-lib` itself and make use of same `ERROR_TYPE`